src/wp-includes/default-filters.php

```diff
@@ -572,6 +572,7 @@
 add_action( 'admin_enqueue_scripts', 'wp_common_block_scripts_and_styles' );
 add_action( 'enqueue_block_assets', 'wp_enqueue_registered_block_scripts_and_styles' );
 add_action( 'enqueue_block_assets', 'enqueue_block_styles_assets', 30 );
+add_action( 'wp_loaded', 'enqueue_block_styles_handle_assets' );
```

Comment:
Does this change will impact on performance?

Reply:
Are we talking about server (PHP) or frontend (resources loading) performances ?

Reply:
Frontend.

Reply:
Currently, if a block style is register with a `style_handle` the CSS file is not included on the frontend. This PR fix that, so it's possible that it'll have an impact in terms of performance.

We could try and mitigate this by dynamically adding a `path` extra data to the dependency to let WordPress inline the style if it's below the threshold.